rasa_core/training/interactive.py

```diff
@@ -792,14 +805,60 @@ def _validate_action(action_name,  # type: Text
     ]
     answers = _ask_questions(questions, sender_id, endpoint)
     if not answers["action"]:
-        corrected_action = _request_action_from_user(predictions, sender_id,
-                                                     endpoint)
-        _correct_wrong_action(corrected_action, endpoint, sender_id,
+        action_name = _request_action_from_user(predictions, sender_id,
+                                                endpoint)
+
+    tracker = retrieve_tracker(endpoint, sender_id,
+                               EventVerbosity.AFTER_RESTART)
+
+    if (tracker.get('active_form', {}).get('name') and
```

Comment:
can we clean up some of these if statements, like make them into functions?

Reply:
the whole thing a function, or only conditions in if statements?

Reply:
the conditions